src/index.js

```diff
@@ -36,7 +36,9 @@ export default class PrismaZoom extends PureComponent {
     // Max delay between two taps to consider a double tap (in milliseconds)
     doubleTouchMaxDelay: 300,
     // Decelerating movement duration after a mouse up or a touch end event (in milliseconds)
-    decelerationDuration: 750
+    decelerationDuration: 750,
+    // Disable zoom on scroll
+    disableScroll: PropTypes.bool,
```

Comment:
I think you misplaced this line. It should go to the `propTypes` object instead, and you should set `disableScroll: false` here.